image_extractor.py

```diff
@@ -230,6 +207,44 @@ def get_data(street_class_names, score_colors):
     return data
 
 
+def extract_pixel_data():
+    """Extract pixel data from all images."""
+    # Read in original data set
+    litter_index = pd.read_csv("data/Litter_Index_Blocks.csv")
+
+    # Iterate through all image data and extract pixels
+    image_folders = os.listdir("image_downloads")
+    for folder in image_folders:
+        print(folder)
+        # Obtain street score for set of images
+        street_score = litter_index[litter_index['OBJECTID'] == int(folder)]\
```

Comment:
NOTE: @monicaaa - The folder path is the same as the OBJECTID. So, we can extract the street_score from the name of the folder.